protocol/x/clob/abci.go

```diff
@@ -88,10 +89,11 @@ func EndBlocker(
 		)
 	}
 
+	deliveredCancels := keeper.GetDeliveredOrderIds(ctx, types.DeliveredCancelKeyPrefix)
```

Comment:
nit: I would probably make two helper functions and abstract away the key prefix from the caller

Reply:
same for the other methods